.kokoro/environment/appengine_flex_container/common.cfg

```diff
@@ -0,0 +1,34 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build logs will be here
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
+
+
+# Specify which tests to run
+env_vars: {
```

Comment:
These ~3 lines seem to be the only different configs between all the environment .cfg files. 

Is it possible to have the different environments share the same base `common.cfg` file.

Reply:
I don't believe so. I see the same copy/paste pattern in the other .kokoro config directories, and the google3 resources seem to point at an individual folder for config files, so I don't think there's a way to share lines.

But I'll double check to see if I can find it documented somewhere before merging to be safe!

Reply:
Yeah I just looked through the kokoro docs again, and it doesn't seem like config inheritance like this is possible (at least from what I could see)